contrib/docker/Dockerfile.builder.fedora

```diff
@@ -30,5 +30,7 @@ RUN wget https://bitcoin.org/bin/bitcoin-core-$BITCOIN_VERSION/bitcoin-$BITCOIN_
     mv bitcoin-$BITCOIN_VERSION/share/man/man1/* /usr/share/man/man1 && \
     rm -rf bitcoin.tar.gz bitcoin-$BITCOIN_VERSION
 
-RUN python3 -m pip install --upgrade pip && \
+RUN python3 -m venv venv && \
+	export PATH=venv/bin:$PATH && \
```

Comment:
Notice that you probably want to make this `PATH` entry absolute. This is important because `lightningd` will change the cwd at startup, hence plugins would attempt to look in a wrong relative path, ultimately falling back to the system-wide `python3` and failing due to missing dependencies.